tests/page-objects/reports/reports.po.js

```diff
@@ -3,8 +3,34 @@ const helper = require('../../helper');
 const itemSummary = '#reports-content .report-body .item-summary';
 const reportBody = '#reports-content .report-body';
 const reportBodyDetails = '#reports-content .report-body .details';
+const datePickerStart = element(by.css('.daterangepicker [name="daterangepicker_start"]'));
+const datePickerEnd = element(by.css('.daterangepicker [name="daterangepicker_end"]'));
+const dateFilter = element(by.css('#date-filter'));
+const reportListID = '#reports-list';
+const list = element(by.css(reportListID));
+const listLoader =  element(by.css(`${reportListID} .loader`));
+const firstReport = element(by.css(`${reportListID} li:first-child`));
+const allReports = element.all(by.css(`${reportListID} li`));
 
 module.exports = {
+  allReports: allReports,
+  firstReport: firstReport,
+  listLoader: listLoader,
+  list: list,
+  reportByUUID: uuid => {
+    return list.all(by.css('li[data-record-id="' + uuid + '"]'));
+  },
```

Comment:
Use templating for readability, and make this a one liner, eg:

```suggestion
  reportByUUID: uuid => list.all(by.css(`li[data-record-id="${uuid}"]`)),
```